resources/qml/Account/AccountWidget.qml

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.7
+import QtQuick.Controls 2.1
+
+import UM 1.4 as UM
+import Cura 1.1 as Cura
+
+Button
+{
+    id: accountWidget
+    property var profile: Cura.API.account.userProfile
+    property var loggedIn: Cura.API.account.isLoggedIn
+
+    implicitHeight: UM.Theme.getSize("topheader").height
+    implicitWidth: UM.Theme.getSize("topheader").height
+
+
+    AvatarImage
+    {
+        id: avatar
+
+
+        width: Math.round(0.8 * parent.width)
+        height: Math.round(0.8 * parent.height)
+        anchors.verticalCenter: parent.verticalCenter
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        source: loggedIn ? profile["profile_image_url"] : UM.Theme.getImage("avatar_default")
+        outlineColor: loggedIn ? UM.Theme.getColor("account_widget_ouline_active") : UM.Theme.getColor("account_widget_ouline_inactive")
```

Comment:
The current lining color is grey, which looks incredibly ugly on a light blue background: 
![screen shot 2018-10-18 at 15 38 26](https://user-images.githubusercontent.com/40687682/47158411-e8390a80-d2eb-11e8-84ac-745d8c8678c0.png)

Changing the lining color to light blue is not an option because it's used in other places. For example border colors of fold-out menus, and we don't want those the be light blue. 

Therefore I believe the Account border should be a separate color.

Reply:
In the current design the top bar has become dark blue, which enables that light grey again. What's more, the lining colour is now the same light grey as the rest of the lining colours, so it might be useful to remove the theme entry again anyway.

Reply:
Currently the lining color is RGB(245, 245, 245) and the one in the designs for the account widget is RGB(229, 229, 229), so it's slightly different. I will let the designers to decide if they want to have the same color for the linings and the account widget outline. What do you think @LukeChen-Ultimaker , @AnneUltimaker and Yi-Ann?

Reply:
Talked with @AnneUltimaker and she agrees on reuse the lining color.